src/drivers/linux_pwm_out/linux_pwm_out.cpp

```diff
@@ -216,7 +216,13 @@ void task_main(int argc, char *argv[])
 		pwm_out = new NavioSysfsPWMOut(_device, _max_num_outputs);
 	}
 
-	if (pwm_out->init() != 0) {
+	/**
+	 * if using pca9685, _fd should large equal than 0 but in pca9685.h, init method return _fd>=0, it's alway be 1
```

Comment:
if the _protocl is "pca9685" and the driver is executed correctly， the return value of "pwm_out->init()" will be nonzero and It will be large than zero.
But in the orignal code , it used  "pwm_out->init()!=0"  to determine whether the driver  initialized correctly. If the ——protocl is "pca9685” that would cause the driver to exit
Now I've change the comment.